src/components/modals/ConfirmDialog.tsx

```
@@ -0,0 +1,31 @@
+}
+
+export function ConfirmDialog({ title, content, actions }: ConfirmDialogProps) {
+  return (
+    <ModalOverlay>
+      <div className="section">
+        <div className="title">{title}</div>
+        <span className="content">{content}</span>
+      </div>
+      <div className="actions">
+        {actions?.map(({ label, onClick }: Action) => (
+          <Button color="primary" variant="transparent" onClick={onClick}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)